village/src/App.js

```diff
@@ -14,11 +17,53 @@ class App extends Component {
   // add any needed code to ensure that the smurfs collection exists on state and it has data coming from the server
   // Notice what your map function is looping over and returning inside of Smurfs.
   // You'll need to make sure you have the right properties on state and pass them down to props.
+  
+  componentDidMount() {
+    this.getSmurfs();
+  }
+
+  getSmurfs() {
+    axios
+    .get(`http://localhost:3333/smurfs`)
```

Comment:
try setting a const variable for url:
`const url = http://localhost:3333/smurfs`

then

```js
axios
  .get(url)
```
It not only makes the code look cleaner, but you can also avoid typos.